CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+This template and our [recommendation pages][website] were made by [RSE]s at
```

Comment:
Even with the link I'd say better to avoid the acronym and use _research software engineer_ rather than RSE.

Reply:
- [ ] Unacronym RSE